### DISCUSSION TITLE
.net 在分布式搜索引擎这块的生态建设远远落后于java。是不是应该想办法解决一下，有没有大牛拉起一个开源项目。

### DISCUSSION THREAD

**Discussion by sgf** [BYJRK/DotNet-Discussions#162] [C# 技术交流]
[搜索引擎](https://db-engines.com/en/ranking/search+engine)

以上链接里面的搜索引擎。不是java就是C++，要么就是rust。

只有C# .net 是空白。

而的架构跨语言注定了无法通吃。

**Comment by woshiwstt50123** [comment]
排名第一的那个Elasticsearch不是支持.net吗？

**Reply by sgf** [reply]
我说的是构建 .net自己的生态,而不是用别人的.SDK.
